<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gluon Game Distibution Wizard</string>
  </property>
  <widget class="QWizardPage" name="apiKeypage">
   <property name="title">
    <string/>
   </property>
   <property name="subTitle">
    <string>Fill in the API Key</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>An API key is a code passed in by computer programs to identify the calling program,its developer,or its user to the web site.You can find the api key for your account in your account settings at gamingfreedom.org</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="apiKeyEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="detailsPage">
   <property name="subTitle">
    <string>Enter basic Game Details</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Game ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Home Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>License</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>160</width>
       <height>162</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="gameIdEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="categoryComboBox"/>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="versionEdit_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="homePageEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="licenseComboBox"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="nextdetailsPage">
   <property name="subTitle">
    <string>Updating Game details, please wait</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Game ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Home Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>License</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>160</width>
       <height>162</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="nextgameIdEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="nextCategory">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nextnameEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nextversionEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="nexthomePageEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="nextlicenseComboBox"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="descriptionPage">
   <property name="subTitle">
    <string>A few words about your game</string>
   </property>
   <widget class="QPlainTextEdit" name="descriptionEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>5</y>
      <width>401</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="subTitle">
    <string>Enter Game Changelog</string>
   </property>
   <widget class="QPlainTextEdit" name="changelogEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>A changelog is a log or record of changes made to a project, such as a website or software project, usually including such records as bug fixes, new features, etc. Most open source projects include a changelog as one of the top level files in their distribution.
</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Thank you for uploading your first game.You can now update game details through
the Distribution Docker whenever you want. You will also be able to see reviews and
comments by people who have played your game.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../../icons/32x32/apps/gluon-uninstall.svg"/>
 </resources>
 <connections/>
</ui>
